packages/grid/_modules_/grid/models/gridExport.ts

```diff
@@ -1,8 +1,34 @@
+/**
+ * Available CSV delimiter characters used to separate fields.
+ * @default ','
+ */
+export type CsvDelimiterCharacter = ',' | ';';
+
 /**
  * The options to apply on the CSV export.
  */
 export interface GridExportCsvOptions {
+  /**
+   * If `true`, the comma character used to separate the integer part from the fractional part of a number.
+   * This flag also changes the delimiter from `,` to `;`.
```

Comment:
typo
```suggestion
   * This flag also changes the delimiter from `,` to `.`.
```

Reply:
It's not a typo. When decimal separator is ',' then delimiter shouldn't also be ',' so it is automatically changed to ';'.